Makefile

```diff
@@ -37,7 +37,7 @@ endif
 
 all: yaml build image
 
-build: chaosdaemon manager chaosfs dashboard dashboard-server-frontend
+build: manager chaosfs dashboard dashboard-server-frontend
```

Comment:
why move out chaosdaemon

Reply:
chaosdaemon is built in `image` target now.